viite-UI/src/utils/backend-utils.js

```diff
@@ -556,6 +562,14 @@
       return self;
     };
 
+    this.withGetRoadLinkById = function (returnData) {
+      self.getRoadLinkById = function (id, callback) {
+        callback(returnData);
+        return returnData;
+      };
+      return self;
+    };
+
```

Comment:
Idea told me that this function is never used. Is it meant to be used somewhere?

Reply:
You are right, it was a remnant of a previous try, deleting it.

Thank you for pointing that out ;)